govcd/edgegateway_lb_service_monitor_test.go

```diff
@@ -0,0 +1,69 @@
+// +build lb lbServiceMonitor functional ALL
```

Comment:
Thanks for expanding the lb to lbServiceMonitor. Let's now quickly decide the format we use for tags with multiple words just to be consistent. Right now we have two "extnetwork" and "lbServiceMonitor":
`// +build extnetwork network functional ALL`
`// +build lb lbServiceMonitor functional ALL`

What's your preference? @dataclouder

Reply:
It's good as it is now

Reply:
Read it's easier with lowerCases, but write with lowercases :-D